### PR TITLE
Update README.md about local env

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This PR tries to clarify the environment setup and changes the section heading to distinguish between Local environment and Running the app locally.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by chuwik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -79,16 +99,7 @@ either by deleting the resource group in the Portal or running `azd down`.
 * [Powershell 7+ (pwsh)](https://github.com/powershell/powershell) - For Windows users only.
   * **Important**: Ensure you can run `pwsh.exe` from a PowerShell command. If this fails, you likely need to upgrade PowerShell.
```
Comment (charris-msft) [inline README.md:100]: ... from a PowerShell terminal. If this fails ...

File: README.md [hunk 2]
```diff
@@ -66,9 +68,27 @@ To reduce costs, you can switch to free SKUs for Azure App Service, Azure Cognit
...
+#### GitHub Codespaces
 
-#### To run locally
+You can run this repo virtually by using GitHub Codespaces, which will open a web-based VS Code in your browser:
+
+[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=599293758&machine=standardLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2)
+
+#### VS Code Remote Containers
+
+A related option is VS Code Remote Containers, which will open the container in your local VS Code:
```
Comment (charris-msft) [inline README.md:85]: "open the container" - which container?
I'm personally not familiar with this

**Review by charris-msft — APPROVED** [review]
I think putting the Codespaces option first will help people get up and running quickly so they can test it out and see if it suits their needs.

File: README.md (hunk 2 above)
Comment (pamelafox) [reply README.md]: Reworded to "A related option is VS Code Remote Containers, which will open the project in your local VS Code using the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers):"

**Review by charris-msft — APPROVED** [review]

**Pull request merged**